### PR TITLE
Node.js: lots of data copied from node-compat-table

### PR DESCRIPTION
**Pull request by jcsahnwaldt**
Data was copied from [node-compat-table](/williamkapke/node-compat-table) (the basis for [node.green](https://node.green/)) using [nct2bcd](/jcsahnwaldt/nct2bcd).

### CODE REVIEW

File: javascript/builtins/Array.json [hunk 1]
```diff
@@ -2255,16 +2288,22 @@
...
-                  {
-                    "type": "runtime_flag",
-                    "name": "--harmony-array-prototype-values"
-                  }
-                ]
-              },
+              "nodejs": [
+                {
+                  "version_added": "6.5.0",
+                  "notes": "<code>--harmony-array-prototype-values</code> is in progress and is not enabled by the <code>--harmony</code> flag.",
```
Comment (Elchi3) [inline javascript/builtins/Array.json]: What does "<code>--harmony-array-prototype-values</code> is in progress" mean?
Also, the node.green table says that this is in 11.0.0 by default, right?

Comment (Elchi3) [reply javascript/builtins/Array.json]: 1. Ah yes, of course, whitespace! Thanks :)
2. Okay, yeah, it's definitely unclear to me why a flag could be "in progress". I'd say "The <code>--harmony-array-prototype-values</code> flag is required; the <code>--harmony</code> flag is not enough in this case.". But then, again, I'm not really a nodejs person :) Let me know what you think of this.
3. Oh interesting, well, I guess we can wait until it is stable. I guess sometimes things regress and are delayed to a later release, so lets not add false promises.

Comment (jcsahnwaldt) [reply javascript/builtins/Array.json]: 1. :-)
2. Yeah, that's probably better. I'll make a new PR after this one is merged.
3. OK.

Comment (Elchi3) [reply javascript/builtins/Array.json]: Cool, merging this then!

**Pull request merged**